packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -843,7 +843,7 @@ class RenderOpacity extends RenderProxyBox {
        super(child);
 
   @override
-  bool get alwaysNeedsCompositing => child != null && (_alpha > 0);
+  bool get alwaysNeedsCompositing => child != null && _alpha > 0;
```

Comment:
https://github.com/flutter/flutter/issues/102179 but we should probably not deal with that in this PR.